support/capabilities/management_cluster_capabilities.go

```diff
@@ -66,12 +71,24 @@ func isGroupVersionRegistered(client discovery.ServerResourcesInterface, groupVe
 
 func DetectManagementClusterCapabilities(client discovery.ServerResourcesInterface) (*ManagementClusterCapabilities, error) {
 	discoveredCapabilities := map[CapabilityType]struct{}{}
+
+	// check for route capability
 	hasRoutesCap, err := isGroupVersionRegistered(client, routev1.GroupVersion)
 	if err != nil {
 		return nil, err
 	}
 	if hasRoutesCap {
 		discoveredCapabilities[CapabilityRoute] = struct{}{}
 	}
+
+	// check for scc capability
```

Comment:
can you please update TestDetectManagementCapabilities to cover this?

Reply:
Updated. https://github.com/openshift/hypershift/pull/846/commits/17d7cd45a2a197b4478808bf2d1cfbd0fdb9a8a2